### DISCUSSION TITLE
运行一段时间好像会崩溃。

### DISCUSSION THREAD

**Discussion by 10531515** [ttttupup/wxhelper#119] [General]
试了好几次，基本上运行半天到一天，微信就突然崩了。不知道哪儿的问题。

**Comment by ttttupup** [comment]
是注入了之后啥都不干，放在那还是有操作。最好提供崩溃日志

**Comment by 10531515** [comment]
注入后，只HOOK了接收消息。其它什么也没干。放置一天大概就会崩溃。
因为日志是一百条日志输出一次，好像不全，是不是我要改成每一条日志输出一次就可以测试了？


另外，崩溃后，有dump内存文件，以及一个崩溃的日志（如附件）。不知道能否有用？
如果用处不大，是不是我应该把日志输出改为一次一保存就可以了？




10531515?????????????
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ttttupup/wxhelper"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年4月28日(星期五) 下午4:23
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ttttupup/wxhelper] 运行一段时间好像会崩溃。 (Discussion #119)





 
是注入了之后啥都不干，放在那还是有操作。最好提供崩溃日志

**Comment by ttttupup** [comment]
有微信的崩溃日志就行了。具体在哪个目录忘了。

**Comment by 10531515** [comment]
[WeChat1bdba8f4.txt](https://github.com/ttttupup/wxhelper/files/11351764/WeChat1bdba8f4.txt)
[WeChat1bdba8f4MidDump.zip](https://github.com/ttttupup/wxhelper/files/11351781/WeChat1bdba8f4MidDump.zip)
上边是崩溃日志的附件。是这个么？

**Comment by ttttupup** [comment]
对，是这个，然后使用的是哪个版本的dll

**Reply by 10531515** [reply]
大约10天前下载的。应该是3.9.2.23。我今天重新看你的代码好像前天更新了一下。我刚刚把更新后的重新编译，正在测试，因为时间还短，还没结果。

**Reply by 10531515** [reply]
10天前的代码好像不全，少了BASE64编码方面的代码，我自己写了个BASE64的放进去编译的。不知道是不是跟这个有关。我今天重新下载的代码是全的，有了BASE64，我再重新测试几天看一下？

**Comment by ttttupup** [comment]
把有问题的dll和日志发出来就行。自己编译的话就更方便了，debug版本就有pdb文件，更方便排查。

**Reply by 10531515** [reply]
好的。之前的DLL没了。因为我刚编译了你最新的代码给旧的覆盖了。我重新把新代码编译为DEBUG版本。然后测试几天，有问题的话。再跟你说下。

**Reply by 10531515** [reply]
好的。之前的DLL没了。因为我刚编译了你最新的代码给旧的覆盖了。我重新把新代码编译为DEBUG版。然后测试几天，有问题的话。再跟着你说下。

**Reply by ttttupup** [reply]
好的，把dll和崩溃日志发一下就可以了。debug版本带pdb的就更好了。

**Comment by hzleii** [comment]
> 试了好几次，基本上运行半天到一天，微信就突然崩了。不知道哪儿的问题。

俺也一样~
